src/features/chart/redux/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const UPDATE_CHART_DATA: "UPDATE_CHART_DATA" = "UPDATE_CHART_DATA";
```

Comment:
You don't need to define a in-line type for the constant, as Typescript can infer it from its assigned value:

```suggestion
export const UPDATE_CHART_DATA = "UPDATE_CHART_DATA";
```